src/LaunchDarkly.Client/FeatureRequestor.cs

```diff
@@ -38,16 +38,23 @@ internal FeatureRequestor(Configuration config)
                     return null;
                 }
                 var flags = JsonConvert.DeserializeObject<IDictionary<string, FeatureFlag>>(content);
-                Logger.LogDebug("Get all flags returned " + flags.Keys.Count + " feature flags");
+
+                Logger.LogDebug("Get all flags returned {0} feature flags",
+                    flags.Keys.Count);
+
                 return flags;
             }
             catch (Exception e)
             {
                 // Using a new client after errors because: https://github.com/dotnet/corefx/issues/11224
                 _httpClient?.Dispose();
                 _httpClient = _config.HttpClient();
-                Logger.LogError("Error getting feature flags: " + Util.ExceptionMessage(e));
-                Logger.LogDebug(e.ToString());
+
+                Logger.LogError(e, 
+                    "Error getting feature flags: {0}",
+                    Util.ExceptionMessage(e));
+
+                Logger.LogDebug("{0}", e);
```

Comment:
I left this, although I disagree with it.  By passing the exception as an argument to LogError, the consumer of this client has the option of whether to log the full exception details or not.

Switched to a replace string, so that for us who don't have Debug log enabled, don't need to call Exception.ToString